packages/flutter/test/services/autofill_test.dart

```diff
@@ -211,9 +211,15 @@ class FakeTextChannel implements MethodChannel {
     incoming = handler;
   }
 
+  @override
+  bool checkMethodCallHandler(Future<void> Function(MethodCall call) handler) => throw UnimplementedError();
```

Comment:
My rationale for not considering this breaking is that this is an unusual test-only implementation that implements instead of extends. I checked the internal repo and all other cases like this use mockito and won't need to be updated.

Reply:
Autofill functionality is relatively very new. I think that is why no tests will be breaking.

Reply:
This wouldn't have broken any tests either, but the class in here is just a pure implementation rather than a mock, so it needed to be updated for this. I don't think this is a particularly common thing to do though.